waddrmgr/manager_test.go

```diff
@@ -1019,47 +920,77 @@ func testImportScript(tc *testContext) bool {
 
 // testMarkUsed ensures used addresses are flagged as such.
 func testMarkUsed(tc *testContext) bool {
-	expectedAddr1 := expectedAddr{
-		addressHash: hexToBytes("2ef94abb9ee8f785d087c3ec8d6ee467e92d0d0a"),
-		used:        true,
+	tests := []struct {
+		name       string
+		typ        addrType
+		in         []byte
```

Comment:
Not a huge deal but we could make this a [20]byte since everything should be a RIPEMD160 hash.